_ZadanieDomowe_8_PokerHands_Base/PokerHands/Services/PokerHandService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using PokerHands.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PokerHands.Services
+{
+    public class PokerHandService : IHandService
+    {
+        public List<Card> GroupCardsFromLowest(IEnumerable<Card> parsedCards) => parsedCards.OrderBy(card => card.Value).ToList();
+
+        public List<Card> GroupByMostOccurences(List<IGrouping<Value, Card>> cardGroupsByValues) =>
+                cardGroupsByValues
+                .OrderByDescending(g => g.Count())
+                .ThenByDescending(g => g.Key)
+                .First()
+                .ToList();
+
+        public IEnumerable<IGrouping<Value, Card>> GroupCardsByValues(List<Card> cardsByValue) =>
+            from card in cardsByValue
+            group card by card.Value
+            into cardGroup
+            select cardGroup;
+
+        public void CheckCorrectnessOfHand(List<Card> parsedCards)
```

Comment:
wygląda dobrze, ale to miało być w subskrybencie, no i dotyczyć całego rozdania, a nie pojedynczej ręki